routes/studentRoutes.js

```diff
@@ -4,7 +4,15 @@ var Student = require(path.join(__dirname, '../models/student'));
 
 routes.GETallStudents = function(req, res) {
   Student.find({}, function(err, allStudents) {
```

Comment:
Oh shoot you are totally right - the reduce method thing fixes all of this - I think we just forgot to take it out in between moving from different ways of doing it